trie/verkle.go

```diff
@@ -50,6 +50,12 @@ func NewVerkleTrie(root verkle.VerkleNode, db *Database, pointCache *utils.Point
 	}
 }
 
+func (trie *VerkleTrie) InsertMigratedLeaves(leaves []verkle.LeafNode) error {
+	return trie.root.(*verkle.InternalNode).InsertMigratedLeaves(leaves, func(hash []byte) ([]byte, error) {
+		return trie.db.diskdb.Get(hash)
+	})
+}
```

Comment:
Just a trampoline to use the new API to avoid magic castings and skipping layers.

For now should be fine... whenever we reconsider https://github.com/gballet/go-verkle/pull/314 maybe this method will go away if it can be as fast as the new API.